packages/api/src/get-repo-orgs.js

```diff
@@ -0,0 +1,97 @@
+
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+// class to receive POSTS with build information
+
+class GetRepoOrgsHandler {
+  constructor (app, client) {
+    this.app = app;
+    this.client = client;
+  }
+
+  // ex: bar -> bas
+  alphabeticallyIncrement (inputStr) {
+    const strFrontCode = inputStr.slice(0, inputStr.length - 1);
+    const strEndCode = inputStr.slice(inputStr.length - 1, inputStr.length);
+    return strFrontCode + String.fromCharCode(strEndCode.charCodeAt(0) + 1);
+  }
+
+  removeDuplicates (results) {
+    const usedIds = new Set();
+    const newResults = [];
+    results.forEach(result => {
+      if (!usedIds.has(result.repoId)) {
+        usedIds.add(result.repoId);
+        newResults.push(result);
+      }
+    });
+    return newResults;
+  }
+
+  // TODO order by flakyness
+  listen () {
+    // return the different environments for a build - all parameters and possible values for next query
+    this.app.get('/api/reposearch', async (req, res, next) => {
```

Comment:
In the spirit of REST, I would probably opt for `/api/repo` for this:

```
GET /api/repo.  -- list of all repos.
GET /api/repo/name -- some specific repo.
POST /api/repo -- create a repo.
PUT /api/repo -- update a repo.
```

We don't have the `POST` and `PUT` endpoints, because this data gets populated by your `/build` endpoint.

Reply:
Fixed, with second route being api/repo/:org/:name